common/buildings/00_tradepost.txt

```diff
@@ -178,288 +178,191 @@ trade_post = {
 		extra_tech_building_start = 10.0 # Never get at start of game
 	}
 	
-	# tr_silk_road_1 = {	# Silk Road Trading Post
-		# potential = {
-			# trade_route = silk_road
-			# silk_road_trade_post_trigger = yes
-		# }
-		# desc = tr_silk_road_1_desc
-		# trigger = {
-			# TECH_TRADE_PRACTICES = 1
-		# }
-		# gold_cost = 200
-		# build_time = 150
-		# ai_creation_factor = 95
-		# add_number_to_name = no
-		
-		# trade_route_wealth = 1	# Affects total value of trade route
-		# tradevalue = 30
-		
-		# extra_tech_building_start = 10.0 # Never get at start of game
-	# }
+	######################
+	#
+	#	Cipher non cultural Trade route buildings
```

Comment:
It's vanilla buildings, this thing not needed here
Imo, it's better to use separate files for mod and vanilla buildings